src/pages/CategoriesPage.js

```diff
@@ -0,0 +1,5 @@
+const CategoriesPage = () => (
+  <h1> Categories </h1>
+);
+
+export default CategoriesPage;
```

Comment:
- [ ] The category should display under construction.

![aaaa](https://user-images.githubusercontent.com/61562626/140954002-ff621d21-f675-4441-b0f1-403648bcc21e.jpeg)